Dockerfile

```
@@ -1,14 +1,16 @@
 ARG OS_VER
 FROM --platform=linux/amd64 nrel/openstudio:$OS_VER as buildstockbatch
 ARG CLOUD_PLATFORM=aws
-
-RUN sudo apt update && sudo apt install -y python3-pip
-RUN sudo -H pip install --upgrade pip
+ENV DEBIAN_FRONTEND=noninteractive
 COPY . /buildstock-batch/
-RUN python3 -m pip install "/buildstock-batch[${CLOUD_PLATFORM}]"
+
+RUN curl -k -LsSf https://astral.sh/uv/install.sh | sh
```

```suggestion
RUN curl -LsSf https://astral.sh/uv/install.sh | sh
```
Avoid insecure curl.